Unrecoverable semantic error(s) found.
m1.module:3:
  Semantic Error: Module imports itself or a subtype: m1
